s64da_benchmark_toolkit/reporting.py

```diff
@@ -95,7 +96,7 @@ def run_report(self, reporting_queue):
             if self.netdata_output_file:
                 netdata.write_stats(self.df, self.netdata_output_file)
 
-        if self.scale_factor:
+        if self.check_correctness:
```

Comment:
`self.scale_factor` is required if `self.check_correctness` is `True`, otherwise it'll not run anyway. So that's why I decided to remove the previous check.

Reply:
This is not critical, but I think it will make the code cleared (clearer intention) if the combined check was done here. I leave it up to you to decide.

Reply:
done